### ISSUE TITLE
Should SimpleNetwork's output be 9 actions?

### ISSUE THREAD

**Issue by YunqiuXu** [valeriechen/ask-your-humans#3]
Hi Valerie,

According to https://github.com/valeriechen/ask-your-humans/blob/master/mazebase-training/train_hierarchy.py#L135
the environment provides 9 actions
However, the output dim of SimpleNetwork is 8
https://github.com/valeriechen/ask-your-humans/blob/master/mazebase-training/train_models.py#L248

Could you help to check this inconsistency? Thanks for your help!

**Comment by valeriechen** [comment]
Yes, in the process of cleaning up the code I accidentally left out the part of the code where I filtered out any 'stop' action so the output dim is 8.
